ESPAsyncWiFiManager.cpp

```diff
@@ -431,14 +441,19 @@ int AsyncWiFiManager::connectWifi(String ssid, String pass) {
   //check if we have ssid and pass and force those, if not, try with last saved values
   if (ssid != "") {
       //trying to fix connection in progress hanging
-      ETS_UART_INTR_DISABLE();
-      wifi_station_disconnect();
-      ETS_UART_INTR_ENABLE();
-
-      WiFi.begin(ssid.c_str(), pass.c_str());
+	  if (WiFi.status() == WL_CONNECTED) {
+		  ETS_UART_INTR_DISABLE();
```

Comment:
hi can you please take care on merging also the ESP32 specific stuff:
https://github.com/alanswx/ESPAsyncWiFiManager/blob/master/ESPAsyncWiFiManager.cpp#L480-L487